p4include/dpdk/psa.p4

```diff
@@ -533,7 +533,8 @@ enum PSA_CounterType_t {
 @noWarn("unused")
 extern Counter<W, S> {
   Counter(bit<32> n_counters, PSA_CounterType_t type);
-  void count(in S index, @optional in bit<32> increment);
+  void count(in S index);
```

Comment:
What does DPDK do when a PSA program calls the original `count` method that does not have a pkt_len parameter?  Nothing?  It would be good to have a comment here if it does anything other than what PSA defines.

Reply:
And also for the corresponding method in the DirectCounter extern.  And for Meter and DirectMeter.